apps/wallet-mobile/src/i18n/LanguageProvider.tsx

```diff
@@ -41,11 +43,12 @@ export const LanguageProvider = ({children}: {children: React.ReactNode}) => {
 
   return (
     <LanguageContext.Provider value={initialState}>
+      {/* @ts-ignore types/react mistmatch */}
       <IntlProvider
         timeZone={timeZone}
         locale={languageCode}
         messages={translations[languageCode]}
-        textComponent={Text}
+        textComponent={Text as any}
```

Comment:
`react-intl` issue, it has a ComponentType | HTML... that is not correctly typed for react-native / react latest version.